helm/istio/charts/grafana/templates/meshclusterrole.yaml

```diff
@@ -8,6 +8,10 @@ metadata:
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
 rules:
+- apiGroups: ["security.openshift.io"]
+  resources: ["securitycontextconstraints"]
+  resourceNames: ["anyuid"]
+  verbs: ["use"]
```

Comment:
Grafana image we switched to is OpenShift-specific and shoudln't need anyuid. I'm looking into what the problem is.